mesonwrap/tempfile.py

```diff
@@ -0,0 +1,29 @@
+# Let's try to make it drop-in replacement for tempfile module
+import platform as _platform
+from tempfile import *
```

Comment:
__[Flake8]__

[F403] 'from tempfile import *' used; unable to detect undefined names


<!-- Comment by [Sider](https://sider.review) -->